CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# cuCIM 21.06.00 (Date TBD)
+# cuCIM [21.06.00](https://github.com/rapidsai/cucim/wiki/release_notes_v21.06.00) (Date TBD)
```

Comment:
For consistency with the changelogs of other RAPIDS libraries, this change should probably be removed. Also, we have a script that updates the changelog during each release, so it would overwrite this change.

Reply:
Oh, I see. Will remove it from here.